src/components/SQForm/SQForm.tsx

```diff
@@ -1,20 +1,20 @@
 import React from 'react';
 import Grid from '@material-ui/core/Grid';
+import {setLocale} from 'yup';
+import type {AnyObjectSchema} from 'yup';
```

Comment:
I need to install a TS imports sorter, but since I haven't, do you mind ensure `type` imports are last? (Unless there's a CSS file, then CSS files go last)

Reply:
Every file will need some attention. We can address that in #649 to avoid bloat in this already massive PR.